code/modules/mob/living/basic/space_fauna/changeling/headslug.dm

```diff
@@ -75,13 +75,9 @@
 /// Just to be super-duper safe to the player, we do return TRUE if all goes well and read that value in check_and_implant() to be nice to the player.
 /mob/living/basic/headslug/proc/infect(mob/living/carbon/victim)
 	var/obj/item/organ/internal/body_egg/changeling_egg/egg = new(victim)
-	egg.Insert(victim)
 
 	egg.origin = mind
 
-	for(var/obj/item/organ/target in src)
-		target.forceMove(egg)
```

Comment:
I think this had some functional effect, for instance to make sure the monkey result of hatching has a human brain and not a monkey one.
If you're removing these lines you'll also want to remove the lines relating to dealing with the organs inside the egg when it hatches.

Reply:
You're right about this, I'm just going to re-add these lines since this PR isn't supposed to change the functionality of the ability at all.

Reply:
why is organ manipulation so doodoo